### PR TITLE
Bump up Cosmos SDK to v0.46.1

### PR DESCRIPTION
**Pull request by taramakage**

### CODE REVIEW

**Review by dreamer-zq — APPROVED** [review]

File: modules/tibc/light-clients/07-tendermint/types/update.go [hunk 1]
```diff
@@ -101,6 +103,10 @@ func checkTrustedHeader(header *Header, consState *ConsensusState) error {
 	// to do this, we check that trustedVals.Hash() == consState.NextValidatorsHash
 	tvalHash := tmTrustedValidators.Hash()
 	if !bytes.Equal(consState.NextValidatorsHash, tvalHash) {
+		fmt.Println(
+			"NextValidatorsHash", hex.EncodeToString(consState.NextValidatorsHash),
+			"ValHash", hex.EncodeToString(tvalHash),
+		)
```
Comment (dreamer-zq) [inline modules/tibc/light-clients/07-tendermint/types/update.go:109]: Remove

File: modules/tibc/apps/mt_transfer/transfer_test.go [hunk 2]
```diff
@@ -42,6 +42,7 @@ func (suite *TransferTestSuite) SetupTest() {
 mt
 A->B B->C
 */
+// TODO
```
Comment (dreamer-zq) [inline modules/tibc/apps/mt_transfer/transfer_test.go:45]: ?

File: modules/tibc/light-clients/07-tendermint/types/update.go (hunk 1 above)
Comment (taramakage) [reply modules/tibc/light-clients/07-tendermint/types/update.go:109]: Resolved

File: modules/tibc/apps/mt_transfer/transfer_test.go (hunk 2 above)
Comment (taramakage) [reply modules/tibc/apps/mt_transfer/transfer_test.go:45]: Reserved for later discussion.

**Review by aofengli — APPROVED** [review]

**Pull request merged**